appengine/flexible/multiple_services/api_gateway/README.md

```diff
@@ -0,0 +1,56 @@
+# Python Google Cloud Microservices Example - API Gateway
+
+This example demonstrates how to deploy multiple python services to [App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/)
+
+## To Run Locally
```

Comment:
@jonparrott I've got it running now so that it will work as previously in production, and the user can run locally by using a `--development` flag from the command line without having to do anything with the code.  Is that sufficient?